### PR TITLE
Repackage - PaloAlto-PAN-OS (MMA to AMA Migration)

### PR DESCRIPTION
**Pull request by v-rusraut**
Required items, please complete
   
   Change(s):
   - Repackaging

   Reason for Change(s):
   - Addition of new PaloAltoPANOS AMA Data Connector

   Version Updated:
   - 3.0.2

   Testing Completed:
   - Done

### CODE REVIEW

**Comment by manishkumar1991** [comment]
@v-rusraut 

Can you provide the screenshot similar to below for this Paloalto and rest of other solution in other PR as well 

image

File: Solutions/PaloAlto-PAN-OS/Hunting Queries/Palo Alto - potential beaconing detected.yaml [hunk 1]
```diff
@@ -58,5 +61,5 @@ entityMappings:
     fieldMappings:
       - identifier: Address
         columnName: IPCustomEntity
-version: 1.0.2
+version: 1.0.3
 kind: Scheduled
```
Comment (manishkumar1991) [inline Solutions/PaloAlto-PAN-OS/Hunting Queries/Palo Alto - potential beaconing detected.yaml:65]: I think ,we cant use this : kind: Scheduled in hunting query template

Comment (v-rusraut) [reply Solutions/PaloAlto-PAN-OS/Hunting Queries/Palo Alto - potential beaconing detected.yaml]: Resolved

**Review by manishkumar1991 — APPROVED** [review]
Approved

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**